rllib/tuned_examples/appo/pendulum_appo.py

```diff
@@ -0,0 +1,48 @@
+from ray.rllib.algorithms.appo import APPOConfig
+from ray.rllib.core.rl_module.default_model_config import DefaultModelConfig
+from ray.rllib.utils.test_utils import add_rllib_example_script_args
+
+parser = add_rllib_example_script_args(
+    default_reward=-300.0,
+    default_timesteps=100000000,
+)
+parser.set_defaults(
+    enable_new_api_stack=True,
+    num_env_runners=4,
+)
+# Use `parser` to add your own custom command line options to this script
+# and (if needed) use their values to set up `config` below.
+args = parser.parse_args()
+
+
+config = (
+    APPOConfig()
+    .environment("Pendulum-v1")
+    .env_runners(
+        num_envs_per_env_runner=20,
+    )
+    .learners(num_learners=1)
```

Comment:
Isn't a single remote learner inefficient?

Reply:
For CPU-only, it's actually better :p 
For 1 GPU, num_learners=0 is better. We have to do some more exploration as to why exactly for 1 CPU, the num_learners=1 setting is better. I'm guessing it has to do with the Learner worker then sharing the same thread/process as the Algo, which could slow down things. For the GPU setup, this is NOT a problem as CUDA is async anyways (can do the forward/backward passes parallel to the CPU).